app/townsquare/views.py

```diff
@@ -142,7 +142,7 @@ def get_sidebar_tabs(request):
             tabs = [new_tab] + tabs
             default_tab = 'grants'
 
-        num_favorites = request.user.favorites.all().count()
+        num_favorites = request.user.favorites.filter(grant=None).all().count()
```

Comment:
Maybe I'm misunderstanding how this works, but how does this filter give you the count of your favorite grants? Shouldn't it be `.exclude(grant=None)`?